Dockerfile

```diff
@@ -42,16 +45,17 @@ ADD supervisor.conf ${SUPERVISORD_DIR}/conf/
 ADD vendor/fix-permissions  /usr/bin/fix-permissions
 RUN chmod +x /usr/bin/fix-permissions
 
-COPY --from=supervisordbuilder /tmp/supervisord ${SUPERVISORD_DIR}/bin/supervisord
+COPY --from=gobuilder /tmp/supervisord ${SUPERVISORD_DIR}/bin/supervisord
 
 ADD assemble-and-restart ${SUPERVISORD_DIR}/bin
-# ADD assemble ${SUPERVISORD_DIR}/bin
-# RUN ${SUPERVISORD_DIR}/bin/assemble
 ADD run ${SUPERVISORD_DIR}/bin
 ADD s2i-setup ${SUPERVISORD_DIR}/bin
 ADD setup-and-run ${SUPERVISORD_DIR}/bin
 
+COPY --from=gobuilder /tmp/getlanguage ${SUPERVISORD_DIR}/bin
+
+ADD language-scripts ${SUPERVISORD_DIR}/language-scripts/
+
 RUN chgrp -R 0 ${SUPERVISORD_DIR}  && \
     chmod -R g+rwX ${SUPERVISORD_DIR} && \
-    chmod -R 666 ${SUPERVISORD_DIR}/conf/* && \
-    chmod 775 ${SUPERVISORD_DIR}/bin/supervisord
```

Comment:
Don't we need to change the permission for supervisord?

Reply:
when its build, it already has the right permissions.